@app/server/src/plugins/CreateUploadUrlPlugin.ts

```diff
@@ -0,0 +1,37 @@
+import { makeExtendSchemaPlugin, gql } from "graphile-utils";
+import uuidv4 from "uuid/v4";
+import * as aws from "aws-sdk";
+import { awsRegion, uploadBucket } from "@app/config";
+
+const CreateUploadUrlPlugin = makeExtendSchemaPlugin(() => ({
+  typeDefs: gql`
+    extend type Mutation {
+      """
+      Get a signed URL for uploading files. It will expire in 60 seconds.
+      """
+      createUploadUrl(contentType: String): String!
+    }
+  `,
+  resolvers: {
+    Mutation: {
+      async createUploadUrl(_query, args, _context, _resolveInfo) {
+        const { contentType } = args;
+        const s3 = new aws.S3({
+          region: awsRegion,
+          signatureVersion: "v4",
+        });
+        const params = {
+          Bucket: uploadBucket,
+          ContentType: contentType,
```

Comment:
This should be heavily validated, I think? But if we use an enum this should be straight forward.